extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1290,7 +1282,8 @@ export class AmpA4A extends AMP.BaseElement {
     let renderPromise = Promise.resolve(false);
     if ((method == XORIGIN_MODE.SAFEFRAME ||
          method == XORIGIN_MODE.NAMEFRAME) &&
-        this.creativeBody_) {
+        this.creativeBody_ &&
+        this.element.getAttribute('type') != 'fake') {
```

Comment:
as discussed offline, pls check if we can get rid of this line

Reply:
removed. Filed bug #13326